addons/website/views/snippets/snippets.xml

```diff
@@ -875,7 +864,25 @@
 
         <we-row string="Font">
             <we-fontfamilypicker data-variable="navbar-font"/>
+        </we-row>
+
+        <we-row string="Format">
             <we-input data-customize-website-variable="null" data-variable="header-font-size" data-unit="px" data-save-unit="rem"/>
+            <we-colorpicker data-variable="header-text-color" data-customize-website-variable=""/>
```

Comment:
![FireShot Capture 1220 - Odoo - Home - My Website - localhost](https://user-images.githubusercontent.com/52911687/191243833-f8facddc-95b6-477d-9d71-b3594718db18.jpg)
Is this what we want? Dropdown items and language selector are not impacted by the option
cc @pierre-pde

Reply:
@bvr-odoo good point.
@qsm-odoo , in master, when you change header colors, it's not updating the submenu dropdown items;
![image](https://user-images.githubusercontent.com/75257428/191247097-6f56f4b7-bfe2-419b-975f-62a2d1b562f8.png)

I think it would be better if the dropdown items would be impacted. What do you think?

Reply:
@qsm-odoo A decision must be taken here. I'm OK with both solution (current behavior and PDE/BVR proposition) I think it's just a matter of taste.. Uber does the same way, [their site](https://www.uber.com) has a black nav bar and the dropdown of its sub menu is white. I guess some users want one solution or the other. If we want to be sure that all users get what they want, there should be an option for that (maybe overkill to do).
![image](https://user-images.githubusercontent.com/78849981/213661597-cf397d46-7c38-42aa-a2e0-adba0cde8d38.png)

Reply:
I'll let @pierre-pde decide of the specs he prefers as it is mainly functional. Although I am not entirely sure of the technical complexity. If you just "change the background color of the dropdown menu", that will never be good: what about link color, hover colors, active color, etc. That's actually why the dropdown are currently forced to white if I am not mistaken: we say "ok the navbar color is set to the present one which has a blue background with red links... but for dropdown-item and dropdown menu reforce the default style so it is not impacted.

So, functional: really a matter of taste, although I think I'd be in favor of having the option: in the orange navbar you showed, I would never want white dropdowns; in the black navbar you showed, I would never want black dropdowns.

Technical: having the dropdown having the same color means more code and more UX complexity and concerns.

So:
- If always white:
    - Problematic functionally IMO (the orange navbar)
    - nothing to do (easy)
- If always the same color:
    - Problematic functionally IMO (the black navbar)
    - More CSS code for the user to load and to maintain (but ok I guess)
    - What about dropdown link color, active state, etc? Probably cannot be automatic in a nice way so config to show to the user? Might be complex?
- Both:
    - Nice functionally
    - But more code
    - Even more complex UX
    
@pierre-pde :muscle:

Reply:
@qsm-odoo  & @Guillaume-gdi 
I don't think I want to offer the options.
IMO, it would be better if dropdowns had same color as navbar.
I think uber website would be a bit uglier but still ok.

FYI, that's how squarespace does it:
https://bit.ly/3XNJ0ih

But white is not the end of the world :)

Reply:
@Guillaume-gdi , however, quick one, I would decrease the size of the 'alignment' dropdown. 
And then re-arrange the items; probably distribute them symetrically. 
![image](https://user-images.githubusercontent.com/75257428/215498077-dc6c7ba3-3b2b-4ebc-9d8d-4c93a7bb5904.png)

Reply:
> re-arrange the items

Done :heavy_check_mark: 
![image](https://user-images.githubusercontent.com/78849981/217504427-e7da1188-b3a7-4d8a-b61a-861be1616aa4.png)